engines/optix/OptiXEngine.cpp

```diff
@@ -88,6 +89,10 @@ void OptiXEngine::_createCameras()
                            0.0635,
                            {"Eye separation"}};
 
+    OptiXContext& context = OptiXContext::get();
+
+    auto camera = std::make_shared<OptiXPerspectiveCamera>();
```

Comment:
well, don't we have only "perspective" camera here? why adding 3 more that use the same program?

Reply:
Yes, they use the same program but each camera has different properties which might not make sense to expose in all camera types. Ideally each camera would use its own program instead of the uber-camera program, but that has to be implement first. So I think it makes sense to keep it this way.

Reply:
I don't see where the perspective camera cuda code does orthographic or panoramic projection. But your choice to keep it like this.

Reply:
Heh, you are right. I assumed the camera types actually worked but apparently not. I removed the other ones so we only have the perspective camera.